src/main/java/org/resitrack/Main.java

```diff
@@ -60,16 +59,51 @@ public static void managePerson(PersonController personController, Scanner scann
             choice = scanner.nextLine();
 
             switch (choice) {
-                case "1":
-                    personController.addNewPerson();
+                case ADD_PERSON:
+                    System.out.println("Adding new person");
+                    boolean personBeAdded = personController.addNewPerson();
+                    if (personBeAdded)
+                        System.out.println("Add success !");
+                    else
+                        System.out.println("Add false !");
```

Comment:
I think we should create a  String variable to hold value of log string and we can make the value reassign depend on boolean value.
 suggestion
![image](https://user-images.githubusercontent.com/69842271/225505616-3fdfedff-ac5f-4c60-9702-6ada7d07350e.png)

Reply:
I applied